index.html

```diff
@@ -1,54 +1,42 @@
-<html lang="en">
-  <head>
-    <title>Guess the Number</title>
-    <link
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
-    />
-    <link rel="stylesheet" type="text/css" href="style.css" />
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Guess The Number</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body class="text-center">
+</head>
+<body class="text-center">
+
     <nav class="navbar navbar-dark bg-dark">
-      <!-- Navigation bar content -->
-      <a class="navbar-brand" href="#">Guess The Number</a>
+        <!-- Navbar content -->
+        <a class="navbar-brand" href="#"><h1>Guess The Number</h1></a>
```

Comment:
- [ ] Try to reduce the banner font size of Guess the Number by 1 or 2. It is too big.